.evergreen/config.yml

```diff
@@ -657,6 +657,28 @@ functions:
           export CSFLE_GIT_REF="${CSFLE_GIT_REF}"
           export CDRIVER_GIT_REF="${CDRIVER_GIT_REF}"
           EOT
+    - command: shell.exec
```

Comment:
non-blocking: subprocess.exec is preferred over shell.exec according to [Evergreen best practices](https://github.com/evergreen-ci/evergreen/wiki/Evergreen-Best-Practices#subprocessexec).

But it might be a pain to rewrite this command so that it works with subprocess.exec, so this is non-blocking 😄

Reply:
I've updated the KMS servers running to use subprocess.exee now.